src/vm/methodtablebuilder.cpp

```diff
@@ -1489,7 +1489,7 @@ MethodTableBuilder::BuildMethodTableThrowing(
         }
     }
 
-#if defined(_TARGET_X86_) || defined(_TARGET_AMD64_) || defined(_TARGET_ARM64_)
+#if defined(FEATURE_HW_INTRINSICS) && (defined(_TARGET_XARCH_) || defined(_TARGET_ARM64_))
```

Comment:
`FEATURE_HW_INTRINSICS` is local to the JIT. I doubt that this ifdef is going to work in VM.